api/RTCTrackEvent.json

```diff
@@ -48,6 +48,55 @@
           "deprecated": false
         }
       },
+      "RTCTrackEvent": {
+        "__compat": {
+          "description": "<code>RTCTrackEvent()</code> constructor",
+          "spec_url": "https://w3c.github.io/webrtc-pc/#dom-rtctrackevent-constructor",
+          "support": {
+            "chrome": {
+              "version_added": "64"
```

Comment:
Yep, in Chrome 63 the `RTCTrackEvent` interface object (and thus constructor) isn't exposed at all.